src/UI/Art/SpriteArtist/SpriteUIUtils.java

```diff
@@ -108,6 +113,51 @@ public static BufferedImage makeTextMenu(Color bg, Color frame, Color focus, Arr
     return menuImage;
   }
 
+  /**
+   * Returns an image with the input string printed within the specified width, in normal text.
+   * @param reqWidth: Actual UI size in pixels that you want to fit the text into.
+   */
+  public static BufferedImage paintTextNormalized(String prose, int reqWidth)
+  {
+    // Find the dimensions of the menu we are drawing.
+    int drawScale = SpriteOptions.getDrawScale();
+    int menuTextWidth = SpriteLibrary.getLettersUppercase().getFrame(0).getWidth() * drawScale;
+    int menuTextHeight = SpriteLibrary.getLettersUppercase().getFrame(0).getHeight() * drawScale;
```

Comment:
It might be more clear to use `charWidth`/`charHeight`. The current name sounds like it might be for the whole menu text.

Reply:
Hah, uncooked pasta. Good catch, though.